src/app/credExplorer/WeightConfig.js

```diff
@@ -175,36 +172,30 @@ class EdgeConfig extends React.Component<{
 }
 
 class NodeConfig extends React.Component<{
-  nodeWeights: NodeWeights,
-  onChange: (NodeWeights) => void,
+  nodeWeights: WeightedNodeType[],
+  onChange: (WeightedNodeType[]) => void,
 }> {
-  render() {
-    const sortedWeights = sortBy(
-      this.props.nodeWeights,
-      ({type}) => type.prefix
-    );
-
-    const controls = sortedWeights.map(({type, weight}) => {
-      const onChange = (value) => {
-        const nodeWeights = this.props.nodeWeights.filter(
-          (x) => x.type.prefix !== type.prefix
+  controls() {
```

Comment:
nit: It is customary to call this `_renderControls` (underscore optional).

Reply:
Done.